doc/user/data/sql_funcs.yml

```diff
@@ -24,16 +28,13 @@
     description: Number of non-_NULL_ inputs.
 
   - signature: jsonb_agg(expression) -> jsonb
-    description: Aggregate values (including nulls) as a jsonb array.
+    description: Aggregate values (including nulls) as a jsonb array. Not recommended for use in materialized views.
```

Comment:
Personal preference to not include this note here and in the other functions. It's already described in the more detailed document pages.